polkadot-protocol/protocol-components/accounts/address-formats.md

```diff
@@ -0,0 +1,183 @@
+```
+
+The address is the concatenated byte series consisting of an address type, the encoded address, and a checksum that is passed into a `Base58` encoder. 
+
+The `Base58encode` function is implemented exactly as defined in Bitcoin and IPFS specifications, using the same alphabet as both of those implementations. 
+
+The `Base58` alphabet eliminates characters that might look ambiguous when printed, for example:
+
+- Non-alphanumerical characters (+ and /)
+- Zero (0)
+- Capital i (I)
+- Capital o (O)
+- Lower-case L (l)
+
+## Address Type
+
+The address type in the `SS58` address format consists of one or more bytes that specify the exact format of the address bytes that follows it.
+
+Currently, the valid values are:
+
+- `00000000b..=00111111b` (0 to 63 inclusive)
+
+    Simple account/address/network identifier. The byte can be interpreted directly as such an identifier.
+
+- `01000000b..=01111111b` (64 to 127 inclusive)
+
+    Full address/address/network identifier. The lower 6 bits of this byte represent the upper 6 bits of a 14-bit identifier, while the next byte defines the lower 8 bits. This allows for identifiers ranging from 0 to 16,383 (2²¹).
+
+- `10000000b..=11111111b` (128 to 255 inclusive)
+
+    This section is reserved for future address format extensions. The address type 42 is designed to be valid across all Polkadot SDK networks that support fixed-length addresses. 
+
+    However, for production networks, using a network-specific version may be preferable to prevent key reuse between networks and mitigate associated issues. 
+
+    By default, Polkadot SDK nodes display keys in address type 42, but Polkadot SDK-based chains with different node implementations—such as those in the Polkadot ecosystem—might use a different default address type.
+
+## Address Length in bytes
+
+There are 16 different address formats, each identified by the total payload length in bytes, which includes the checksum.
+
+| Total | Type | Raw account | Checksum |
+|-------|------|-------------|----------|
+| 3     | 1    | 1           | 1        |
+| 4     | 1    | 2           | 1        |
+| 5     | 1    | 2           | 2        |
+| 6     | 1    | 4           | 1        |
+| 7     | 1    | 4           | 2        |
+| 8     | 1    | 4           | 3        |
+| 9     | 1    | 4           | 4        |
+| 10    | 1    | 8           | 1        |
+| 11    | 1    | 8           | 2        |
+| 12    | 1    | 8           | 3        |
+| 13    | 1    | 8           | 4        |
+| 14    | 1    | 8           | 5        |
+| 15    | 1    | 8           | 6        |
+| 16    | 1    | 8           | 7        |
+| 17    | 1    | 8           | 8        |
+| 35    | 1    | 32          | 2        |
+
+## Checksum Types
+
+Polkadot SDK offers several potential checksum strategies, each providing varying length and longevity guarantees. 
+
+There are two types of checksum preimages (`SS58` and `AccountID`) along with a range of checksum lengths from 1 to 8 bytes.
```

Comment:
It would nice to link these (`SS58` and `AccountID`), unfortunately I couldn't find this right away in the codebase.. so not exactly sure what it is referring to.

Reply:
I'll add this comment to the sheet and try to find it in code base, same result from my side. I can't find the right code to link here

Reply:
I'm pretty sure it is here, but I am not completely certain. Perhaps we can merge this, and find and add it later? 

https://github.com/paritytech/polkadot-sdk/blob/3313163485c4d4946ea97477407107796469f1c4/substrate/primitives/core/src/crypto.rs#L277